discord/voice_client.py

```diff
@@ -45,6 +45,7 @@
 import struct
 import threading
 from typing import Any, Callable, List, Optional, TYPE_CHECKING, Tuple, Union
+from typing_extensions import Unpack
```

Comment:
This is not a runtime requirement so it will fail for most people. We usually import from `typing_extensions` purely at type checking time inside `TYPE_CHECKING`. That being said, this style of unpacking kwargs is *not* used in the repository at the moment. Originally the reason was because it was not formally accepted into Python but it was added during [PEP-692](https://peps.python.org/pep-0692/). However, last time we attempted to incorporate this the ergonomics were not ideal since it had a lot of bugs in Pyright and did not have good auto complete experience.

I am unsure if this has changed since then.